src/Gui/MainWindow.cpp

```diff
@@ -346,6 +346,17 @@ MainWindow::MainWindow(QWidget * parent, Qt::WindowFlags f)
 
     // accept drops on the window, get handled in dropEvent, dragEnterEvent
     setAcceptDrops(true);
+
+    // setup font substitutions for NaviCube
+    // Helvetica usually gives good enough results on mac & linux
+    // in rare cases Helvetica matches a bad font on linux
+    // Nimbus Sans Narrow and Open Sans Condensed added as fallback
+    // Bahnschrift is a condensed font available on windows versions since 2017
+    // Arial added as fallback for older version
+    auto substitutions = QStringLiteral("Bahnschrift,Helvetica,Nimbus Sans Narrow,Open Sans Condensed,Arial,Sans");
```

Comment:
very minor: we use spaces after comma

Reply:
Here would be a bad idea. This isn't parameter list. ;)

Reply:
the comma is a separator to later split this into a QStringList, if I add spaces the second QString in the list will become `" Helvetica"` (with a leading space) instead of just `"Helvetica"` no? I'm not sure what effect that would have

Reply:
OK then. To avoid that somebody changes this , I recommend to add a comment.

Reply:
I will add it, although I think it would be strange for any dev to change the contents of a harcoded string without considering the implications.

Reply:
I feared the auto-formatters But sure, they should not change a string value. So let's keep it as it is without a comment.
Sorry for the noise.